### ISSUE TITLE
[Bug] Crashing when opening a new window in an external app

### ISSUE THREAD

**Issue by NotWoods** [mozilla-mobile/fenix#7922]
## Steps to reproduce
Open a link with `target="_blank"` in a PWA or a custom tab or a TWA

### Expected behavior
The window opens

### Actual behavior
App crashes because no activity is set up to handle the new window

### Crash information

https://sentry.prod.mozaws.net/operations/fenix/issues/7137984/

thrown by `CustomTabWindowFeature$start`:
```
No Activity found to handle Intent { act=android.intent.action.VIEW cat=[mozilla.components.pwa.category.SHORTCUT] dat= pkg=org.mozilla.fenix (has extras) }
```

**Comment by NotWoods** [comment]
I can't reproduce this crash :/

When I take the same steps as in the stack trace, I can open a window with no issues.

**Comment by pocmo** [comment]
Migration build: https://sentry.prod.mozaws.net/share/issue/5cabe213fa8044639e74fd80c702f554/

**Comment by ValentinTimisica** [comment]
I think this is the same as #8412, so #8903 with [A-C PR](https://github.com/mozilla-mobile/android-components/pull/6122) should fix this crash.

**Comment by ValentinTimisica** [comment]
Verified as fixed on #8412.
